src/transforms.jl

```diff
@@ -211,7 +211,7 @@ end
 # IMPLEMENTATIONS
 # ----------------
 
-include("transforms/colspec.jl")
+include("colspec.jl")
```

Comment:
This include should go in the main TableTransforms.jl file. This list of includes here is for implementations of the Transform trait.

Reply:
ok

Reply:
Made in [933754d](https://github.com/JuliaML/TableTransforms.jl/pull/38/commits/933754de0e18e04bafc02872229966abf01cbfa8)